Sources/UILocalNotification+Sweet.swift

```diff
@@ -24,9 +24,9 @@ public extension UILocalNotification {
 
     public static func find(_ id: String) -> UILocalNotification? {
         let notifications = UIApplication.shared.scheduledLocalNotifications ?? [UILocalNotification]()
-        let notification = notifications.filter { $0.userInfo?[UILocalNotification.idKey] as? String == id }
+        let filteredNotifications = notifications.filter { $0.userInfo?[UILocalNotification.idKey] as? String == id }
```

Comment:
Sometimes I have trouble with using `notification in` though, it won't recognise the type this way 🤔